src/pages/home/HomePage.js

```diff
@@ -40,46 +29,23 @@ import {fetchCountryCodeByRequestIP} from '../../libs/actions/GeoLocation';
 import KeyboardShortcut from '../../libs/KeyboardShortcut';
 import {redirect} from '../../libs/actions/App';
 import RightDockedModal from '../../components/RightDockedModal';
-import withWindowDimensions, {windowDimensionsPropTypes} from '../../components/withWindowDimensions';
-import compose from '../../libs/compose';
 import {getBetas} from '../../libs/actions/User';
 import Account from '../../libs/actions/Account';
 
 const propTypes = {
-    isSidebarShown: PropTypes.bool,
     network: PropTypes.shape({isOffline: PropTypes.bool}),
-    ...windowDimensionsPropTypes,
 };
+
 const defaultProps = {
-    isSidebarShown: true,
     network: {isOffline: true},
 };
 
-class HomePage extends Component {
+class HomePage extends PureComponent {
```

Comment:
Are we using PureComponents now?

Reply:
Yeah, I think we always were, but will probably see them often in the react native stuff since performance bottlenecks are more obvious on mobile.